src/pages/accomodation/components/ImagesList.tsx

```diff
@@ -0,0 +1,84 @@
+import { Box, ImageList, ImageListItem, useMediaQuery, useTheme } from '@mui/material';
+import ShowPhotos from '@src/components/full-view-accommodation/full-view-accommodation';
+import Slider from '@src/components/shared/Slider';
+import { Media } from '@src/types/accommodation.types';
+import { renderedImageType } from '@src/types/accommodationImages.types';
+import { useState } from 'react';
+import { getImageSources } from '../utils/imagesListUtils';
+
+export default function ImagesList({ imagesData }: { imagesData: Media[] }) {
+  const [openDialog, setOpenDialog] = useState(false);
+
+  const handleOpenDialog = () => {
+    setOpenDialog(true);
+  };
+  const handleCloseDialog = () => {
+    setOpenDialog(false);
+  };
+
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down('md'));
+
+  const [activeStep, setActiveStep] = useState(0);
+
+  const images = imagesData.slice(0, 5).map((image, index) => ({
```

Comment:
`5` looks like magic number, create a constant for it

Reply:
5 is number of images which will be displayed on single accommodation page